tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py

```diff
@@ -252,7 +253,8 @@
         Field(
             item="65B",
             name='RECEIVE_SSI',
-            friendly_name="receives SSI",
+            friendly_name="Receives Disability Benefits: SSI Under Title XVI-SSI or " +
```

Comment:
@victoriaatraft @elipe17 please confirm if this change is the correct one or the one located in [line 574](https://github.com/raft-tech/TANF-app/blob/880f13b5032f0e178ea45c1e9831279654434fcb/tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py#L574)